Library/Homebrew/github_packages.rb

```diff
@@ -138,10 +138,8 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, dry_run:)
     repo = "homebrew-#{repo}" unless HOMEBREW_OFFICIAL_REPO_PREFIXES_REGEX.match?(repo)
 
     version = bottle_hash["formula"]["pkg_version"]
-    rebuild = if (rebuild = bottle_hash["bottle"]["rebuild"]).positive?
-      ".#{rebuild}"
-    end
-    version_rebuild = "#{version}#{rebuild}"
+    rebuild = bottle_hash["bottle"]["rebuild"]
+    version_rebuild = "#{version}#{"-#{rebuild}" if rebuild.positive?}"
```

Comment:
This was implemented in https://github.com/Homebrew/brew/pull/10977 (and this PR needs rebased now that is merged)

Reply:
I've rebased this PR.